bowtie/_cli.py

```diff
@@ -765,6 +765,16 @@ def _check_fail_fast_provided(
     return value
 
 
+def _validate_schema_and_dialect(
+    schema: Any,
+    dialect: Dialect,
+) -> bool:
+    schema = schema["$schema"]
+    if str(dialect.uri) != schema:
```

Comment:
Have another look at what I mentioned about `if foo: return True`

Reply:
okay, so we need to reduce the size of the code as much as possible.